src/main/java/io/unlogged/logging/perthread/RawFileCollector.java

```diff
@@ -278,11 +278,11 @@ public void run() {
                         errorLogger.log(e);
                     }
 
-                    if (networkClient != null && !"localhost-token".equals(networkClient.getToken())) {
+                    if (networkClient != null && !"".equals(networkClient.getServerUrl())) {
```

Comment:
does this need additional check for string "null"?

Reply:
If networkClient is null, the `networkClient.getServerUrl()` would cause a NPE.

Reply:
oh, okay will add a check for "null" also.